src/components/ProductListContainer/index.js

```diff
@@ -14,7 +15,7 @@ const fakeProductList = [
     occupancyPercentage: 50,
     groupCapacity: 10,
     suscriptorsNumber: 5,
-    expirationDate: new Date().toLocaleTimeString(),
+    expirationDate: '2021-04-04T16:00:00.000Z',
```

Comment:
For these fake dates you could use `dayjs` to add days/hours to the current date and time to get dates that are always interesting to test and display:

`dayjs().add(7, 'day').toISOString()`